src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -429,7 +433,7 @@ private void finalizeCurrentVariant(final List<VariantContext> unmergedCalls,
 
         // apply VQSLod-based filters
         VariantContext filteredVC =
-                noVqslodFilteringRequested ? genotypedVC : filterSiteByVQSLOD(genotypedVC, vqsLodMap, yngMap, performGenotypeVQSLODFiltering);
+                noVqslodFilteringRequested ? genotypedVC : filterSiteAlleleByVQSLOD(genotypedVC, vqsLodMap, yngMap, performGenotypeVQSLODFiltering);
```

Comment:
Technically this is filtering at the site level (ie the entire site gets filtered, not a site-allele).  If you wanted to make it more specific, maybe `filterSiteByAlleleSpecificVQSLOD` would be more accurate